staging/2019/measures-data-quality.json

```diff
@@ -2475,7 +2475,7 @@
     }
   },
   {
-    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps \r- Avoidance of Inappropriate Use",
+    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps\\r\\n \r- Avoidance of Inappropriate Use",
```

Comment:
@lamroger This came from the latest csv file from Sarah, is this something we should fix before merge?

Reply:
@KyleApfel It looks like it actually come's from `util/measures/2019/quality-measures.csv`. It had an additional `\r\n`. Once removed, it reverted back to the original `\r` which I am keeping because this is how it is for 2018. If I need to change it, let me know.